addons/medical_treatment/functions/fnc_treatment.sqf

```diff
@@ -147,6 +147,8 @@ if (_callbackProgress isEqualTo {}) then {
 
 [_medic, _patient, _bodyPart, _classname, _itemUser, _usedItem] call _callbackStart;
 
+[QGVAR(treatmentStartedLocal), [_medic, _patient, _bodyPart, _classname, _itemUser, _usedItem]] call CBA_fnc_localEvent;
```

Comment:
These don't need to have `Local` at the end of them, that is implied by `localEvent` and is documented.

It looks like these should also be simplified to `ace_treatmentStarted` rather than `ace_medical_treatment_treatmentStartedLocal`, but I'd let someone else confirm that

Reply:
Quite a lot of events in ACE are QGVAR. Are they supposed to be internal then?

Reply:
- Updated `ace_treatmentSucceded` with `_itemUser` and `_usedItem` parameters;
- Renamed other two events to `ace_treatmentStarted` and `ace_treatmentFailed`

Reply:
Yes I believe we decided at some point that QGVAR events for internal only and public events can just use `ace_`, probably not documented anywhere though